spec/rspec/support/output_formatter_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'spec_helper'
+require 'rspec/support/output_formatter'
+require 'rspec/matchers/fail_matchers'
+
+module RSpec
+  module Support
+    describe OutputFormatter, ".format" do
```

Comment:
None of the specs you have in this example group actually call `OutputFormatter.format`.  They are all just using the `eq` matcher logic that is still in rspec-expectations (but will later be refactored to use `OutputFormatter.format`).

The specs should be re-written to use `OutputFormatter.format`, particularly since that's the new code in this PR.